### ISSUE TITLE
 Training method

### ISSUE THREAD

**Issue by lihui52** [houliang428/CNN-for-face-anti-spoofing#2]
Hi, I don't seem to see the training strategies as the paper in the code.

**Comment by houliang428** [comment]
Yes. I am not repeating the work in the paper. First it's only a two-class classification problem. Second,after cropping, about 80% part of image is human face. So there is not too much noise (like background information).What we want to process is only face image, so the less background information it contains, the better my model will perform. Third, I changed the parameters in the fully-connected layers, since I am intended to design a lightened CNN. Thus it does not require too many iterations for the model to come to convergence. I set my mini batch to 8 and epochs to 4000. And it showed better performance than the method proposed in the paper.
